src/Manifest/ManifestSchema.php

```
@@ -9,85 +9,191 @@
             }
 
-            if (!isset($entry['mode']) || !is_string($entry['mode'])) {
+            if (!is_string($modeValue)) {
                 throw new RuntimeException(
-                    sprintf('Manifest entry for "%s" is missing a "mode" key.', $destination),
+                    sprintf('Manifest "modes" value for pattern "%s" must be a string.', $pattern),
                 );
             }
 
-            $mode = FileMode::tryFrom($entry['mode']);
+            $mode = FileMode::tryFrom($modeValue);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'yii\scaffold\Manifest\FileMode' in method 'validateModes'. (undefined)

(StaticAccess)